### ISSUE TITLE
Aliases with :replace-deps do not report updates

### ISSUE THREAD

**Issue by jr0cket** [liquidz/antq#44]
#### Description :octocat:
Aliases that include a dependency via the `:replace-deps` key are not checked for newer versions.  Tested with the 0.9.2 version of antq

> The `:replace-deps` key is often used for defining tooling aliases that do not need to use the dependencies from the main project.

#### Reproduction guide :beetle:
Add an alias to the user level configuration that adds the antq library via the `:replace-deps` key, using a version less than the current, e.g. 0.9.1
```
  :project/outdated
  {:replace-deps {antq/antq {:mvn/version "0.9.1"}}
   :main-opts  ["-m" "antq.core"]}
```

Run the clojure command with the alias
```
clojure -M:project/outdated
```

*Observed behaviour:* :eyes: :broken_heart:
antq reports no library updates available, even though 0.9.2 is available.

*Expected behaviour:* :heart: :smile:
Updating the alias to use `:extra-deps` and run the command again.  This will report a new version of antq/antq is available
```
  :project/outdated
  {:extra-deps {antq/antq {:mvn/version "0.9.1"}}
   :main-opts  ["-m" "antq.core"]}
```

```
|    :file |                       :name | :version | :latest-version |
|----------+-----------------------------+----------+-----------------|
| deps.edn |                   antq/antq |    0.9.1 |           0.9.2 |
```

#### System Info :computer:
- Clojure CLI tools: "1.10.1.727"
- Java: openjdk version "1.8.0_275"
- OS: Ubuntu linux 20.10

**Comment by liquidz** [comment]
@jr0cket Thanks for your reporting!

Here may be the cause.
https://github.com/liquidz/antq/blob/0.9.2/src/antq/dep/clojure.clj#L38
Adding `:replace-deps` should work, but I have no time for now. 
So I'll try tonight.

**Comment by jr0cket** [comment]
Oh, it is very simple to add, so I created a PR after testing it on my every growing  https://github.com/practicalli/clojure-deps-edn configuration.

I added a :hack/antq alias to test it locally, using a local root config
```
  :hack/antq
  {:replace-deps
   {antq/antq {:local/root "/home/practicalli/projects/clojure/community-tools/antq"}}
   :main-opts ["-m" "antq.core"]}
```

**Comment by liquidz** [comment]
@jr0cket Just released v0.9.3
